apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/BillingBreakdown.constants.ts

```diff
@@ -45,6 +45,14 @@ export const BILLING_BREAKDOWN_METRICS: Metric[] = [
     category: 'Authentication',
     unitName: 'MAU',
   },
+  {
+    key: PricingMetric.MONTHLY_ACTIVE_THIRD_PARTY_USERS,
+    name: 'Monthly Active Third-Party Users',
+    units: 'absolute',
+    anchor: 'mauSso',
```

Comment:
**question:** Haven't checked the concept of "anchors", but is "...Sso" correct?